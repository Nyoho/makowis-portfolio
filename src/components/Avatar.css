:local .avatar {
    border-radius: 50%;
}