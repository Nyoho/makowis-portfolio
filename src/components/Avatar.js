// @flow
import React from 'react';
import { avatar as className } from './Avatar.css';

type Props = {
  src: string,
  alt: string,
};

// alt は受け取っているが認識しないのでdisableにする
// eslint-disable-next-line jsx-a11y/alt-text
const Avatar = (props: Props) => <img {...{ ...props, className }} />;

export default Avatar;
