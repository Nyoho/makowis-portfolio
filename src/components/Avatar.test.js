/* eslint-env jest */
// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import Avatar from './Avatar';

it('renders Avatar', () => {
  const props = { src: 'http://example.com/hoge.png', alt: 'Example' };
  const tree = renderer.create(<Avatar {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
