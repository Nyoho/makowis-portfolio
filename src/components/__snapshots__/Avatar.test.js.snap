// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Avatar 1`] = `
<img
  alt="Example"
  className={undefined}
  src="http://example.com/hoge.png"
/>
`;
